### ISSUE TITLE
NavBar: link does not take the full height

### ISSUE THREAD

**Issue by knutkj** [OfficeDev/Office-UI-Fabric#476]
`ms-NavBar-link` does not take height of `ms-NavBar`. You have to hit the text, not above or under, to be able to navigate with the anchor. It is only `19px` in height, and not touch friendly.

Screen shot:

![image](https://cloud.githubusercontent.com/assets/691727/14231670/2b25d756-f98c-11e5-9733-4fcde2fde9ee.png)

Sample code to illustrate:

```html
<div class="ms-NavBar" style="background-color: magenta;">
  <div class="ms-NavBar-openMenu js-openMenu">
    <i class="ms-Icon ms-Icon--menu"></i>
  </div>
  <div class="ms-Overlay"></div>
  <ul class="ms-NavBar-items">    
    <li class="ms-NavBar-item">
      <a
         class="ms-NavBar-link"
         style="background-color: cyan;"
         onclick="alert('clicked')">
        Home
      </a>
    </li>
  </ul>
</div>
```

Try it here: https://jsfiddle.net/ks6fadgo/

**Comment by mikewheaton** [comment]
On the [documentation for NavBar](http://dev.office.com/fabric/components/navbar) you'll see that we've actually attached a click event to `ms-NavBar-item` rather than `ms-NavBar-link`. This gives it a larger tap/click area.

Given that it's used for navigation I'd expect developers to use `ms-NavBar-link` and I think we should update our sample (and fix the CSS) so that this works well. Thanks for reporting the issue.

**Comment by mikewheaton** [comment]
Fixed by #615. NavBar has been merged with CommandBar, which has a full-height button or link.
